github-for-jira.sd.yml

```diff
@@ -647,6 +647,75 @@ environmentOverrides:
             Period: 300
             ComparisonOperator: GreaterThanThreshold
             Statistic: Maximum
+
+      - type: sqs
+        name: backfill_dlq
```

Comment:
Hi @joshkay10 sorry I am bit lost. Can you tell me is it going to create new sqs queue on prod (since the whole block a new block) ? And how does it connect to the backfill as dlq?

Reply:
Micros automatically provision the dead letter queues since we set MaxReceiveCount
This is just attaching an alarm to those queues, it is confusing since you can't see the dlq creation in here and the name of them i had to go get from AWS directly :(